### ISSUE TITLE
Add functionality to customer_database.py

### ISSUE THREAD

**Issue by AppleBlu** [devkristaps/Mortgage-Calculator#47]
We want the function `register_customer()` to be more user friendly. At the moment when the user inputs one of the inputs incorrectly it will ask them to restart and close the program. If the user messed up on the password input it would mean they would have to restart and retype all of their previous inputs that they have already done correctly, this would be frustrating. Instead when they make a mistake with an input, we want it to just take them back to the input they made a mistake with. So if they mess up the input username, instead of restarting the whole program it should just get them to re enter username until it is correct. This should be done for every input except for the first one `user_first_name`.
This might be tricky, hopefully you can do it.

**Comment by devkristaps** [comment]
Blocked by https://github.com/devkristaps/Mortgage-Calculator/issues/50

**Comment by devkristaps** [comment]
I would say this would be better if we could just go for what we discussed, user enters all the data and when he enters the last password, program asks if this is correct information, if yes, continue, if no, redo the registration process again.
